corehq/apps/app_manager/dbaccessors.py

```diff
@@ -540,6 +540,21 @@ def get_case_types_from_apps(domain):
     return set(q.run().aggregations.modules.case_types.keys) - {''}
 
 
+@quickcache(['domain'])
+def get_app_languages(domain):
+    # unique terms aggregation is intentionally avoided
+    #   to reduce memory footprint of the query, as the number of
+    #   apps/builds in a domain should be low enough to agg in Python
+    result = (AppES()
+            .domain(domain)
+            .source('langs')
+            .run().hits)
+    langs = set()
+    for lang_list in result:
+        langs = langs.union(set(lang_list['langs']))
```

Comment:
not sure which one is more expensive
```
langs = list(map(lambda x: set(x['langs']), result))
return set().union(*langs)
```
was just curious about this approach, fine to not change.

Reply:
they both have the same cost since they are doing the same thing

Reply:
Cool, i just thought if the single call to `union` with all lists would make a difference.
+ Less lines of code

Reply:
Not a hard preference, but I like the current one, since it's slightly more readable (I feel) with no performance difference.